### DISCUSSION TITLE
Some docs/readme clarifications needed

### DISCUSSION THREAD

**Discussion by bogdanailincaipnt** [vitejs/vite-plugin-react-swc#55] [Ideas]
Hello!
I am interested in starting to use this package. However there are a few things that are not clear to me:

1) In the [Vite docs](https://vitejs.dev/plugins/#vitejs-plugin-react-swc), it says that this package "uses esbuild during build". Do they refer to the prod build here ? And does that mean that Rollup will no longer be used for prod build if I use this package?
2) The README says "_tsconfig is not resolved, so properties other than the ones listed above behaves like TS defaults_" in the [Caveats](https://github.com/vitejs/vite-plugin-react-swc#caveats) section. From what I tested, at least some config options from my `tsconfig.json` are still working in development. So I am probably missing something.

Thanks in advance for your responses!

**Comment by ArnaudBarre** [comment]
Yes here build refers to the build command that always bundle with rollup, but the transformation of modules is done via esbuild when not using plugins, or swc + esbuild when using plugins (I need to update the doc).

I confirm that tsconfig is not read. Which options are you referring too? I think they just match the default of the plugin.

**Reply by bogdanailincaipnt** [reply]
Ok, I see, thank you. Yes, a docs update would be nice. You are right on the tsconfig. It was my mistake.
